src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -50,8 +52,12 @@ const ItemUser = ({photo, username, followed, onPress, userid}) => {
         </View>
         <View style={styles.containerButton}>
           <Pressable style={styles.followAction(32, 32)}>
-            <Animated.View style={{position: 'absolute', opacity: followIconFadeAnimation}}>
-              <MemoIc_Checklist width={32} height={32} />
+            <Animated.View style={{position: 'absolute', top: 4, opacity: followIconFadeAnimation}}>
+              <IconChecklist
+                width={32}
+                height={32}
+                color={isAnon ? COLORS.anon_primary : COLORS.signed_primary}
+              />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change here correctly applies the color condition to the `IconChecklist` component based on the `isAnon` prop. This is a good use of conditional rendering.
```diff
- <MemoIc_Checklist width={32} height={32} />
+ <IconChecklist
+   width={32}
+   height={32}
+   color={isAnon ? COLORS.anon_primary : COLORS.signed_primary}
+ />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->